lib/ui/platform_dispatcher.dart

```diff
@@ -1290,7 +1291,11 @@ class PlatformConfiguration {
 class ViewConfiguration {
   /// A const constructor for an immutable [ViewConfiguration].
   const ViewConfiguration({
-    this.window,
+    FlutterView? view,
+    @Deprecated('''
```

Comment:
This is missing the version in which this was deprecated, see https://github.com/flutter/flutter/wiki/Tree-hygiene#deprecations.

Reply:
Got it, thanks for sharing. I need to bookmark that page.

This should resolve this comment 40922b5c8b5b029fe619801b8435d1aa37b8d926.